src/Joomlatools/Console/Command/Database/Install.php

```diff
@@ -140,11 +157,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
     public function check(InputInterface $input, OutputInterface $output)
     {
+        echo "TARGET DIR ".$this->target_dir;
```

Comment:
@levnikmyskin There is still an `echo` statement here :)

Reply:
Sorry, removed now :D